core/unix/signal_linux_aarch64.c

```diff
@@ -257,12 +257,12 @@ sigcontext_to_mcontext_simd(priv_mcontext_t *mc, sig_full_cxt_t *sc_full)
                         memcpy(&mc->simd[i].q, &fpc->vregs[i], sizeof(mc->simd->q));
                     }
                     for (i = 0; i < MCXT_NUM_SVEP_SLOTS; i++) {
-                        memcpy(&mc->svep[i].u32,
+                        memcpy(&mc->svep[i].u16,
                                (byte *)sve + SVE_SIG_PREG_OFFSET(quads_per_vector, i),
-                               sve->vl);
+                               sve->vl / 8);
```

Comment:
So this used to copy 8x more data than it needed?

Reply:
Yes it did.